messages/run-common.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+	"flags": {
+		"formatDescription": "specify results output format",
+		"formatDescriptionLong": "Specifies output format with results written directly to the console.",
+		"normalizesevDescription": "return normalized severity 1 (high), 2 (moderate), and 3 (low), and the engine-specific severity",
+		"normalizesevDescriptionLong": "Returns normalized severity 1 (high), 2 (moderate), and 3 (low), and the engine-specific severity. For the html option, the normalized severity is displayed instead of the engine severity.",
```

Comment:
@teresa-allen-sfdc , preference between the version in the file and the following?
```"Returns normalized severity 1 (high), 2 (moderate), and 3 (low) and the engine-specific severity. For the html option, normalized severity is displayed instead of the engine severity."```